framework/src/org/apache/cordova/SplashScreenPlugin.java

```diff
@@ -127,8 +129,11 @@ private void setupSplashScreen(SplashScreen splashScreen) {
 
         // auto hide splash screen when custom delay is defined.
         if (autoHide && delayTime != DEFAULT_DELAY_TIME) {
-            Handler splashScreenDelayHandler = new Handler();
-            splashScreenDelayHandler.postDelayed(() -> keepOnScreen = false, delayTime);
+            Handler delayHandler = new Handler();
+            delayHandler.postDelayed(
+                () -> delayHandlerCondition ? keepOnScreen = false : null,
```

Comment:
This is causing a build to fail

```
/home/runner/work/cordova-android/cordova-android/framework/src/org/apache/cordova/SplashScreenPlugin.java:134: error: incompatible types: lambda body is not compatible with a void functional interface
                () -> delayHandlerCondition ? keepOnScreen = false : null,
> Task :CordovaLib:compileDebugJavaWithJavac
```

I don't really understand Java's lambda syntax but I think it might be trying to return a value, when it expects a `void` return value.